docs/4.1/admin-guide.md

```diff
@@ -2081,10 +2081,14 @@ teleport:
   storage:
       # The region setting sets the default AWS region for all AWS services 
       # Teleport may consume (DynamoDB, S3)
-      region:  us-east-1 
+      region: us-east-1 
 
       # Path to S3 bucket to store the recorded sessions in.
-      audit_sessions_uri:  s3://Example_TELEPORT_S3_BUCKET/records
+      audit_sessions_uri: "s3://Example_TELEPORT_S3_BUCKET/records"
+
+      # Teleport assumes credentials. Using provider chains, assuming IAM role or 
+      # standard .aws/credentials in the home folder. 
+      access_key: 
```

Comment:
this line should be removed as well.

Reply:
Good call, I've removed it in the last commit.